data/successors/successor side missions.txt

```diff
@@ -101,7 +101,7 @@ mission "Successors: Predecessor Research 1"
 			label who
 			action
 				set "Successors: who"
-			`	"Our Predecessors were our now-gone ancestors. Until the time of fifteen hundred years ago, they ruled this space and much beyond as part of their own empire. Many and their bloodlines died as much of the empire fell to ruin, but some made passage through the wormhole to the cluster that we, as their Successors, now call our home."`
+			`	"Our Predecessors were our now-gone ancestors. Until the time of fifteen hundred years ago, they ruled this space and much beyond as part of their own empire. Many of their bloodlines died as much of the empire fell to ruin, but some made passage through the wormhole to the cluster that we, as their Successors, now call our home."`
```

Comment:
I think this is intended. It means that many people died along with their families/bloodlines (or more likely, their entire House).

The dialog does say at some point that many houses died out in this era, and I think this refers to the same thing. I would like to ask @Daeridanii1 to confirm it, though.

Reply:
That makes sense, and (especially in the King James-esque English the Successors use) it's totally valid. I would be tempted to rephrase to "many, and many bloodlines" in that case, but I'm notoriously pedantic about such things :smile:

Reply:
Maybe "Many perished along with their bloodlines", or something similar?

Reply:
I like that phrasing as well!

Reply:
```suggestion
			`	"Our Predecessors were our now-gone ancestors. Until the time of fifteen hundred years ago, they ruled this space and much beyond as part of their own empire. Many perished along with their bloodlines as much of the empire fell to ruin, but some made passage through the wormhole to the cluster that we, as their Successors, now call our home."`
```

Reply:
Yes, like that.